controller/inviteController.js

```diff
@@ -0,0 +1,121 @@
+const Invite = require('../model/inviteModel');
+const HttpStatus = require('http-status-codes');
+const ObjectId = require('mongoose').Types.ObjectId;
+
+const ValidationUtil = require('../util/validationUtil');
+const StatusEnum = require('../model/enums').StatusEnum;
+
+const create = (req, res) => {
+
+    const user = req.decoded;
+
+    //Only admins can create invite objects
+    if (!user || !ValidationUtil.isUserAdmin(user.accessType)) {
+        return res.status(HttpStatus.FORBIDDEN).json({
+            status: StatusEnum['ERROR'],
```

Comment:
At the previous meeting we agreed to remove the status property from the response object, so you might as well do it straight away. Less refactoring later.